distribution/docs/src/main/jdocs/content/_plugins/content-uri-access-plugin-contents.adoc

```diff
@@ -0,0 +1,16 @@
+:type: plugin
+:status: published
+:title: Content URI Access Plugin
+:link: _content_uri_access_plugin
+:plugintypes: access
+:summary: Prevents a Metacard's resource URI from being overridden by an incoming UpdateRequest.
+
+The Content URI Access Plugin prevents a Metacard's resource URI from being overridden by an incoming UpdateRequest.
+
```

Comment:
Is it worth saying something about why we have the content uri plugin? We don't want someone to update an a metacard they have access to with a resource URI that points to a resource that is classified above their level.